extensions/wikia/TemplateDraft/TemplateDraft.setup.php

```diff
@@ -57,3 +57,9 @@
  * Right rail module
  */
 $wgAutoloadClasses['TemplateDraftModuleController'] = $IP . '/skins/oasis/modules/TemplateDraftModuleController.class.php';
+
+/**
+ * Add approvedraft action (?action=apprevedraft)
+ */
+$wgAutoloadLocalClasses['ApprovedraftAction'] = 'extensions/wikia/TemplateDraft/ApprovedraftAction.php';
```

Comment:
Should be `$wgAutoloadClasses['ApprovedraftAction'] = __DIR__ . '/ApprovedraftAction.php';` as normal.